app/components/gate.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import Loading from 'react-loading';
```

Comment:
I actually just grabbed it cause `react-loading` was already a direct dependency of `blip`. It looks like it is actually used in the `Header` (for the print view loading spinner), but I do agree that consolidation is for the best and we should trend toward that, so I'll switch out the dependency here.

Reply:
Oh, you're right.  Forgot about in the Header.  I'm fine with even a comment suggesting we look at consolidating to one or the other, rather than forcing the use of the `viz` Loader right now.  Up to you, really

Reply:
Part of me is thinking maybe we should retire the `viz` component instead.  Thoughts?

Reply:
Ah, I see you've got it done already.  I'm good with that 👍